styles/ratings.css

```diff
@@ -0,0 +1,47 @@
+@import url(/styles/index.css);
+
+.rating-wrapper {
+  cursor: pointer;
+}
+
+.rating-wrapper label {
+  display: inline-block;
+  position: relative;
+  width: 2rem;
+}
+
+.rating-wrapper label:before {
+  content: "\2729";
+  position: absolute;
+  display: inline-block;
+  top: 0;
+  right: 0;
+  width: 50rem;
+  font-size: 20px;
```

Comment:
can use relative units above 16px

Reply:
Thanks for bringing it to notice, I have changed it now